users/migrations/0002_add_users_test.py

```diff
@@ -0,0 +1,39 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('users', '0001_initial'),
+    ]
+
+    def generate_user_test_data(apps, schema_editor):
+        from django.contrib.auth.models import User
+
+        # Create a super user
+        superuser = User()
+        superuser.is_active = True
+        superuser.is_superuser = True
+        superuser.is_staff = True
+        superuser.username = 'admin'
+        superuser.email = 'admin@gmail.com'
+        superuser.set_password('admin1234')
+        superuser.save()
+        # Create normal users
+        users_test_data = [
+            ("AlonShakaroff", "Alon", "Shakaroff", "test$1234", "alon@gmail.com", "profile_pics/alon.jpg"),
+            ("OmerCohenShor", "Omer", "Cohen-Shor", "test$5678", "omer@gmail.com", "profile_pics/omer.jpg"),
+            ("TomerNewman", "Tomer", "Newman", "test$8910", "tomer@gmail.com", "profile_pics/tomer.jpeg"),
+            ("YuliSuliman", "Yuli", "Suliman", "test$3643", "yuli@gmail.com", "profile_pics/yuli.jpg"),
+            ("MatanPeretz", "Matan", "Peretz", "test$8979", "matan@gmail.com", "profile_pics/matan.jpg")
```

Comment:
Is this the correct path? looks like the pics are under the media folder

Reply:
> Is this the correct path? looks like the pics are under the media folder

Yes, in the ResuMe/settings.py, the MEDIA_ROOT is the media directory.
![image](https://user-images.githubusercontent.com/36789773/161562851-06564764-8228-4eb5-98e8-0c3ab7c4f2f3.png)